include/torch-mlir/Dialect/Torch/Utils/Utils.h

```diff
@@ -19,6 +19,15 @@ namespace Torch {
 int64_t toPositiveDim(int64_t dim, int64_t inputRank);
 bool isValidDim(int64_t dim, int64_t inputRank);
 bool getListConstructElements(Value v, SmallVectorImpl<Value> &elems);
+constexpr StringRef getDefaultGeneratorGlobalVarName() {
```

Comment:
This should be a detail of backends. It shouldn't be part of the Torch Dialect.

Reply:
Removed all the related function from include/torch-mlir/Dialect/Torch/Utils/Utils.h.